### PR TITLE
More clear process description.

### PR DESCRIPTION
**Pull request by lprot**

### CODE REVIEW

File: apps/backup [hunk 1]
```diff
@@ -64,11 +64,15 @@ B2NAND() {
 	if [ "$1" != "-b" ]; then
 		if [ ! -f $NAND/$MUVERSION-RCC_fs0.bin ]; then
 			if [[ $SIZE -gt 150000 ]]; then
-				on -f mmx cp -r $BACKUPFOLDER/$MUVERSION-RCC_fs0.bin $NAND/ 2>> $LOG
-				if [[ "$TRAINVERSION" != *MHI2Q* ]]; then
-					on -f mmx cp -r $BACKUPFOLDER/$MUVERSION-MMX_fs0.bin $NAND/ 2>> $LOG
+				if [ -f $BACKUPFOLDER/$MUVERSION-RCC_fs0.bin ]; then
```
Comment (harman-f) [inline apps/backup:67]: that should never happen, as RCC is checked and created just before as part of the backup

File: apps/flash [hunk 2]
```diff
@@ -184,38 +184,38 @@ PF=$VOLUME"/patches/"$TRAINVERSION"_"$MUVERSION"_PATCH" # patch folder
 echo -ne "\n" >> $LOG
 
 if [[ "$TRAINVERSION" = *MHIG* ]]; then
-	echo -ne "--- Read data from patch DB\n" | $TEE -i -a $LOG
 	if grep -E "RCC.*ifs-root.*App" $TMP/version.txt >/dev/null; then >> $LOG
+		echo -ne "--- Reading patch DB...\n" | $TEE -i -a $LOG
 		DB_PARSER
 	else
-		echo -ne "--- Patch not found in DB\nPlease get in contact with M.I.B team and share your backup\n" | $TEE -i -a $LOG
+		echo -ne "--- Error! Info about ifs-root partition is missing.\nPlease get in contact with M.I.B team and share your backup\n" | $TEE -i -a $LOG
```
Comment (harman-f) [inline apps/flash:191]: replace --- with !!!

File: apps/backup (hunk 1 above)
Comment (lprot) [reply apps/backup:67]: It happens because at installation only basic backup is done.

File: apps/flash (hunk 2 above)
Comment (lprot) [reply apps/flash]: Fixed.

**Pull request merged**